eclipse-pass/pass-support
# 679 fix nihms loader its

Fixes the ITs to use the new `pass-data-client`. 

Things to note:

- All ITs assertions remained intact except for the `TransformAndLoadSmokeIT` assertion on the submissions, which changed from 45 to 37. Discussed with Mark and this is something that we will look into farther once we test data regression after the data migration. Reference: https://github.com/eclipse-pass/pass-support/pull/41/commits/955add57cd160ce210cd9e195212061d088fb138 and noted in ticket: https://github.com/eclipse-pass/main/issues/679#issuecomment-1710424657
- Removed the attempt retries that were required for `ElasticSearch` and the assertions that were intended to ensure a test object was persisted. They are no longer needed since we don't need to wait for `ElasticSearch` to index.
- Removed `GrantNormalizationIT` - This was for the `ElasticSearch` normalization. This coverage is now in the `NihmsPassClientServiceIT`


To test: `Mvn verify`

Smoke and Load IT should return in the logs:

For Compliant records:
`INFO 18:01:45.748 (NihmsCsvProcessor) 9 records were processed with 0 failures.`

For In process records:
`INFO 18:02:26.262 (NihmsCsvProcessor) 19 records were processed with 0 failures.`

For the Non-compliant records:
`INFO 18:03:04.030 (NihmsCsvProcessor) 20 records were processed with 0 failures.`

Add the end of all files:
`INFO 18:03:04.031 (NihmsTransformLoadService) Transform and load complete. Processed 3 files`
